components/TopNav.tsx

```diff
@@ -1,15 +1,27 @@
 import React, { useEffect, useState } from "react";
 import GitHubButton from "react-github-btn";
+import HomeButton from "./HomeButton";
+import { useRouter } from 'next/router';
 
 const TopNav = () => {
+  const router = useRouter();
   const [windowWidth, setWindowWidth] = useState<number>(0);
   useEffect(() => {
     setWindowWidth(window.innerWidth);
   }, []);
 
   return (
     <div className="h-16 mb-10 w-full px-3 border-b-2 border-slate-700 text-white flex justify-between items-center">
-      <div className="flex items-center flex-col w-1/2">{/* empty div */}</div>
+      <div className="flex items-center flex-col w-1/2">
+        {
+          router.pathname !== "/" ?
```

Comment:
I think you can do it this way
```suggestion
          router.pathname !== "/" &&
```